build.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# rmm build script
+
+# This script is used to build the component(s) in this repo from
+# source, and can be called with various options to customize the
+# build as needed (see the help output for details)
+
+# Abort script on first error
+set -e
+
+NUMARGS=$#
+ARGS=$*
+
+# Get root of git repository without assuming location of build.sh script
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPT_DIR"
+REPODIR=$(git rev-parse --show-toplevel)
+
+VALIDARGS="clean librmm rmm -v -g -n -h"
+HELP="$0 [clean] [librmm] [rmm] [-v] [-g] [-n] [-h]
+   clean  - remove all existing build artifacts and configuration (start over)
+   librmm - build and install the librmm C++ code and librmm_cffi
```

Comment:
I'd remove `and librmm_cffi` from the text for `librmm`.

Reply:
Fixed